packages/fx-core/src/component/developerPortalScaffoldUtils.ts

```diff
@@ -122,6 +122,10 @@ async function updateManifest(
   }
   const existingManifestTemplate = manifestRes.value;
 
+  if (!existingManifestTemplate) {
+    return err(new ObjectIsUndefinedError("manifest.json downloaded from template"));
```

Comment:
should it be localized?

Reply:
I don't think we do localization here usually. the whole message is something like this " error.ObjectIsUndefinedError": "Object %s is undefined, which is not expected" where we put object name.